### PR TITLE
Add `getOption` method

### PR DESCRIPTION
**Pull request by SentryMan**
Add `getOption` method for processor option retrieval.

### CODE REVIEW

File: prism-core/src/main/java/io/avaje/prism/internal/APContextWriter.java [hunk 1]
```diff
@@ -448,7 +458,6 @@ public static void write(PrintWriter out, String packageName, boolean moduleRead
             + "            .toString()\n"
             + "            .replaceFirst(id, \"\")\n"
             + "            .replaceFirst(\"/classes/java/main\", \"\")\n"
-            + "            .replaceFirst(\"/classes/java/test\", \"\")\n"
```
Comment (rbygrave) [inline prism-core/src/main/java/io/avaje/prism/internal/APContextWriter.java:451]: Why was this removed? Is this a bug fix or just not necessary?

Comment (SentryMan) [reply prism-core/src/main/java/io/avaje/prism/internal/APContextWriter.java:451]: oh woops

**Review by rbygrave — APPROVED** [review]
